gordo_components/data_provider/azure_utils.py

```diff
@@ -20,8 +20,7 @@ def get_datalake_token(
     Parameters
     ----------
     interactive
-        If true then fall back to interactive authentication in case
-        `dl_service_auth_str` is empty
+        If true then use interactive authentication
```

Comment:
While you're in here,could you add type defs to the docs? `interactive: bool` for example

Reply:
and Fixed

Reply:
I guess also type hint to the actual method? like 
`def get_datalake_token(
    interactive:bool =True, dl_service_auth_str:str = None
    interactive:bool =False, dl_service_auth_str:str = None
) -> lib.DataLakeCredential:`